aws/resource_aws_ssm_parameter_test.go

```diff
@@ -486,6 +516,20 @@ resource "aws_ssm_parameter" "test" {
 `, rName, pType, value)
 }
 
+func testAccAWSSSMParameterBasicConfigTypeUpdated(rName, pType, value string) string {
```

Comment:
Nit: This test configuration function looks identical to `testAccAWSSSMParameterBasicConfig` so will consolidate these on merge. 👍